src/open_deep_research/cli.py

```
@@ -0,0 +1,199 @@
+    Returns:
+        Configuration dictionary
+    """
+    if config_path is None:
+        return {}
+    
+    config_path = Path(config_path)
+    if not config_path.exists():
+        logger.warning(f"Configuration file {config_path} does not exist")
+        return {}
+    
+    with open(config_path, "r") as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)